ci/cases/yamls/build.yaml

```diff
@@ -0,0 +1,3 @@
+builds:
+ - gefs: './build_all.sh'
+ - gfs: './build_all.sh -g'
```

Comment:
I think we need the `-u` back when expanding back to the full suite of tests instead of the two that were used for the smoke test.

Reply:
Yes and you can add an abratry number of systems by adding a new system name to the Jenkins file in the build matrix.  We can take a stab at the[ Dynamic Matrix](https://plugins.jenkins.io/dynamic-axis/) Plug-tin